README.md

```diff
@@ -18,14 +18,12 @@
 <!-- [![Gitter](https://badges.gitter.im/DeepCTR/community.svg)](https://gitter.im/DeepCTR/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) -->
 
 
-DeepCTR is a **Easy-to-use**,**Modular** and **Extendible** package of deep-learning based CTR models along with lots of
+DeepCTR is a **Easy-to-use**, **Modular** and **Extendible** package of deep-learning based CTR models along with lots of
 core components layers which can be used to easily build custom models.You can use any complex model with `model.fit()`
 ，and `model.predict()` .
 
-- Provide `tf.keras.Model` like interface for **quick experiment**
-  . [example](https://deepctr-doc.readthedocs.io/en/latest/Quick-Start.html#getting-started-4-steps-to-deepctr)
-- Provide  `tensorflow estimator` interface for **large scale data** and **distributed training**
-  . [example](https://deepctr-doc.readthedocs.io/en/latest/Quick-Start.html#getting-started-4-steps-to-deepctr-estimator-with-tfrecord)
+- Provide `tf.keras.Model` like interface for **quick experiment**. [example](https://deepctr-doc.readthedocs.io/en/latest/Quick-Start.html#getting-started-4-steps-to-deepctr)
```

Comment:
`like interface` -> `like interfaces` or `like an interface` ?

Reply:
‘like interfaces ’ is good

Reply:
> `like interface` -> `like interfaces` or `like an interface` ?

What needs to be done for my submitted pr to be merged. My submission is able to support python3.9